analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -120,7 +120,7 @@ class Bundler : PackageManager() {
                     declaredLicenses = declaredLicenses.toSortedSet(),
                     aliases = emptyList(),
                     vcs = VcsInfo.EMPTY,
-                    vcsProcessed = processProjectVcs(workingDir),
+                    vcsProcessed = processProjectVcs(projectDir = workingDir, homepageUrl = homepageUrl),
```

Comment:
Using named parameters here seems a bit verbose.

Reply:
I did not want to provide a value for the middle `vcsFromProject` parameter, hence the named arguments. Do you prefer passing explicitly passing `VcsInfo.EMPTY` for `vcsFromProject`?

Reply:
No, I missed that `processProjectVcs` takes three arguments. Actually naming the first parameter would not be required, but I leave this up to you.

Reply:
Ok, I've removed the first named arguments.